Library/Formula/freexl.rb

```diff
@@ -1,9 +1,11 @@
-require 'formula'
-
 class Freexl < Formula
-  homepage 'https://www.gaia-gis.it/fossil/freexl/index'
-  url 'http://www.gaia-gis.it/gaia-sins/freexl-sources/freexl-1.0.0g.tar.gz'
-  sha1 '2a5b1d3ebbaf217c7bda15b5b3f1e0222c6c1502'
+  homepage "https://www.gaia-gis.it/fossil/freexl/index"
+  url "https://www.gaia-gis.it/gaia-sins/freexl-sources/freexl-1.0.1.tar.gz"
+  sha256 "df0127e1e23e9ac9a189c27880fb71207837e8cba93d21084356491c9934b89b"
+
+  option "without-check", "Skip compile-time make checks."
+
+  depends_on "doxygen" => [:optional, :build]
```

Comment:
Perhaps given we usually build docs by default, might be worth bumping it to ` :recommended `, but I'm indifferent either way. Just wrote it up like that because of the murdering conditional deps thing :smile_cat: